packages/web/src/components/nav/desktop/AccountSwitcher/AccountListContent.tsx

```diff
@@ -67,15 +71,22 @@ export const AccountListContent = ({
         backgroundColor='white'
         borderBottom='default'
         alignItems='center'
-        justifyContent='flex-start'
+        justifyContent='space-between'
         ph='l'
-        pv='s'
+        pv={navBackElement ? 'l' : 's'}
```

Comment:
I really can't decide if we'd be better off doing a second implementation that composes a lot of the same pieces with a different header.

I'm more and more wary of these use-case-specific styling overrides for each instance instead of having variants that you select via a single prop and letting the component internally decide how everything should be laid out and styled. As it is, if `navBackElement` changes content or size, it might break this and require that we change the padding value here or add another condition to check, etc.

This is probably okay for now, but if we need to change it much more, I'd like to revisit it.

Reply:
agree!